src/Uno.UI/UI/Xaml/DependencyObjectStore.cs

```diff
@@ -265,6 +265,37 @@ private void Dispose(bool disposing)
 
 			ValidatePropertyOwner(property);
 
+			// As a performance optimization, avoid creating the property details and try get the value in this fast path.
+			if ((propertyDetails ??= _properties.FindPropertyDetails(property)) is null)
```

Comment:
Interesting optimization! Let's see what the build says.

Reply:
@Youssef1313  Looks like the build is globally passing, though there are [some unit tests](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=95848&view=ms.vss-test-web.build-test-results-tab&runId=2531982&resultId=101303&paneView=debug) that are not passing.

Reply:
@Youssef1313 could you take a look at those tests? Thanks!

Reply:
@jeromelaban I bypassed this optimization for attached properties as that's all what the failing tests about.

Reply:
@Youssef1313 so you'd need to change the test, is that what you mean?

Reply:
I didn't change the test. Instead, I modified the code to avoid the new optimization when the property is attached. However, looks like some other tests are still failing :/